packages/shared/src/graphql/posts.ts

```diff
@@ -189,6 +189,7 @@ export const POST_BY_ID_QUERY = gql`
       trending
       content
       contentHtml
+      pinnedAt
```

Comment:
Does this fix this issue? https://dailydotdev.slack.com/archives/C01L0QXQJNB/p1729159495038119

Reply:
No I decided not to fix it here, it probably needs some bigger refactoring 🙏